### PR TITLE
Fix deprecated: useStrict without argument

### PR DESCRIPTION
**Pull request by didierfranc**

### CODE REVIEW

**Comment by zalmoxisus** [comment]
Hey, @didierfranc.

Thanks for the contribution! The first one is a good catch, but the second one should be replaced with `isStrictModeEnabled` which is intended to replace deprecated `useStrict` without argument

File: src/spy.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ const monitors = {};
 const scheduled = [];
 
 function configure(name, config = {}) {
-  if (typeof config.onlyActions === 'undefined') onlyActions[name] = mobx.useStrict();
+  if (typeof config.onlyActions === 'undefined') onlyActions[name] = mobx.useStrict(true);
```
Comment (zalmoxisus) [inline src/spy.js]: Should be 
```js
if (typeof config.onlyActions === 'undefined') onlyActions[name] = mobx.isStrictModeEnabled && mobx.isStrictModeEnabled();
```

**Comment by didierfranc** [comment]
Done 😉

**Comment by zalmoxisus** [comment]
Awesome. Thanks a lot! Will cut a release tomorrow.

**Comment by didierfranc** [comment]
Before this merge mobx-remotedev was broken by the last version of mobx 😉

**Pull request merged**